test/cpp/aoti_inference/test.cpp

```diff
@@ -144,6 +144,8 @@ void test_aoti_package_loader_multi_gpu(
     const std::string& device,
     bool use_runtime_constant_folding) {
   torch::NoGradGuard no_grad;
+  // Ensure that this test will reset the default CUDA device on exit.
+  torch::DeviceGuard device_guard(c10::Device("cuda"));
```

Comment:
This was an unrelated fix; this `multi_gpu` test ended with the device selection being set to the last available GPU index, which caused all subsequent tests to segfault for me locally.